src/rollback_interface.cpp

```diff
@@ -44,7 +44,7 @@ RollbackNode::RollbackNode(Map *map, v3s16 p, IGameDef *gamedef)
 	NodeMetadata *metap = map->getNodeMetadata(p);
 	if (metap) {
 		std::ostringstream os(std::ios::binary);
-		metap->serialize(os);
+		metap->serialize(os, 1); // FIXME: version bump??
```

Comment:
I didn't want to bother, but it looks like this really doesn't include a serialization ver